python/ray/_private/usage/usage_constants.py

```diff
@@ -8,16 +8,22 @@
 
 USAGE_STATS_ENABLED_ENV_VAR = "RAY_USAGE_STATS_ENABLED"
 
-USAGE_STATS_ENABLED_MESSAGE = (
+USAGE_STATS_ENABLED_FOR_CLI_MESSAGE = (
     "Usage stats collection is enabled. To disable this, add `--disable-usage-stats` "
     "to the command that starts the cluster, or run the following command:"
     " `ray disable-usage-stats` before starting the cluster. "
     "See https://docs.ray.io/en/master/cluster/usage-stats.html for more details."
 )
 
+USAGE_STATS_ENABLED_FOR_RAY_INIT_MESSAGE = (
+    "Usage stats collection is enabled. To disable this, run the following command:"
+    " `ray disable-usage-stats` before `ray.init()`. "
```

Comment:
Users might not have ray.init() in the script, since we do implicit init. For clarity, reword this to

```
Usage stats collection is enabled. To disable, run `ray disable-usage-stats` prior to starting Ray.
```

Here and below.